### ISSUE TITLE
Support for AWS Glue Job Notification Delay

### ISSUE THREAD

**Issue by Zambonilli** [terraform-providers/terraform-provider-aws#11829]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Add support for the AWS Glue Job Notification Delay After feature. This feature allows for AWS Glue to send a notification to Clouwatch rules when a job takes longer than the configured minutes value. To configure this property it looks like we need to include a "NotificationProperty" that is of type object with 1 field called "NotifyDelayAfter" that is the minute integer value. It is indicated on the AWS CLI as `--notification-property <value>`.

### New or Affected Resource(s)

* aws_glue_job

### Potential Terraform Configuration

Example possible implementation of the terraform for the aws_glue_job.
```hcl
resource "aws_glue_job" "example" {
  name     = "example"
  role_arn = "${aws_iam_role.example.arn}"

  command {
    script_location = "s3://${aws_s3_bucket.example.bucket}/example.py"
  }
  notification_property {
    notify_delay_after = 90
  }
}

```

### References

* [NotificationProperty](https://docs.aws.amazon.com/glue/latest/webapi/API_NotificationProperty.html)
* [AWS CLI Create Glue Job](https://docs.aws.amazon.com/cli/latest/reference/glue/create-job.html)
* [AWS Glue Create Glue Job API](https://docs.aws.amazon.com/glue/latest/webapi/API_CreateJob.html)

**Comment by jhole89** [comment]
Anyone got an issue with me having an initial go at this?

**Comment by bflad** [comment]
Support for this functionality has been merged and will release with version 2.50.0 of the Terraform AWS Provider, end of next week. Thanks to @jhole89 for the implementation. 👍

**Comment by Zambonilli** [comment]
Thank you so much for the quick turn around here. I was hoping to get a PR together this weekend to support this feature but you guys already have it in production!
